### ISSUE TITLE
The bazel build seems to be broken 

### ISSUE THREAD

**Issue by Arseny-N** [deepmind/multidim-image-augmentation#24]
Hello, currently I try to build the package with 

```
mkdir -p /tmp/build_output_${USER}
docker run \
		  -e USER="$(id -u)" \
		  -u="$(id -u)" \
		  -v ${PWD}:/src/workspace \
		  -v /tmp/build_output_${USER}:/tmp/build_output \
		  -w /src/workspace \
		  l.gcr.io/google/bazel:latest \
		  --output_user_root=/tmp/build_output test --python_version=py3 --config=nativeopt //...
```

I get the following output:

```
Extracting Bazel installation...
Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
...
```

It goes on indefinitely, if I kill it I get the following:

```
^C
Bazel caught interrupt signal; shutting down.

INFO: Call stack for the definition of repository 'org_tensorflow' which is a http_archive (rule definition at /tmp/build_output/a08c2e4811c846650b733c6fc815a920/external/bazel_tools/tools/build_defs/repo/http.bzl:292:16):
 - /src/workspace/WORKSPACE:26:1
WARNING: Download from https://mirror.bazel.build/github.com/tensorflow/tensorflow/archive/v1.13.1.tar.gz failed: class com.google.devtools.build.lib.bazel.repository.downloader.UnrecoverableHttpException GET returned 404 Not Found
ERROR: An error occurred during the fetch of repository 'org_tensorflow':
   java.io.IOException: thread interrupted
ERROR: build interrupted
INFO: Elapsed time: 21.933s
^CINFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded)
ERROR: Couldn't start the build. Unable to run tests
FAILED: Build did NOT complete successfully (0 packages loaded)
```

The link described in the error message (https://mirror.bazel.build/github.com/tensorflow/tensorflow/archive/v1.13.1.tar.gz) is indeed dead. 

What could be the cause ?

**Comment by gourie** [comment]
Same problem here. Any update on the above? Are there any plans to support Tf2.x?

**Comment by ardila** [comment]
+1 It would be great if there were working build instructions

**Comment by ardila** [comment]
bazel test --python_version=py3 --config=nativeopt //...WARNING: Download from https://mirror.bazel.build/github.com/tensorflow/tensorflow/archive/v1.13.1.tar.gz failed: class com.google.devtools.build.lib.bazel.repository.downloader.UnrecoverableHttpException GET returned 404 Not FoundERROR: error loading package '': in /home/diego_captionhealth_com/src/multidim-image-augmentation/workspace.bzl: in /home/diego_captionhealth_com/.cache/bazel/_bazel_diego_captionhealth_com/68d194adb5809d1fecd401c49d83adb1/external/org_tensorflow/tensorflow/workspace.bzl: Label '@org_tensorflow//third_party:nccl/nccl_configure.bzl' is invalid because '@org_tensorflow//third_party/nccl' is a subpackage; perhaps you meant to put the colon here: '@org_tensorflow//third_party/nccl:nccl_configure.bzl'?INFO: Elapsed time: 0.225sINFO: 0 processes.FAILED: Build did NOT complete successfully (0 packages loaded)FAILED: Build did NOT complete successfully (0 packages loaded)


And then

bazel test     --incompatible_disable_genrule_cc_toolchain_dependency=false     --incompatible_disable_legacy_cc_provider=false     --incompatible_disable_third_party_license_checking=false     --incompatible_no_transitive_loads=false     --incompatible_bzl_disallow_load_after_statement=false     --incompatible_disallow_load_labels_to_cross_package_boundaries=false     --config=nativeopt //...ERROR: Unrecognized option: --incompatible_disable_genrule_cc_toolchain_dependency=false

**Comment by ardila** [comment]
I think I fixed this. I have published a pip installable package that should work for tf 1.5 or tf 2.0 here

https://pypi.org/project/multidim-image-augmentation/0.0.1/

**Comment by ardila** [comment]
Built using this repo
https://github.com/ardila/custom-op

**Comment by joth76** [comment]
@ardila that's great step forward Diego! The extensive commit history on https://github.com/ardila/custom-op/commits/master really highlights just how big a delta has built up between the published (from Google internal) and public-buildable version, e.g. delta in public TF2 support, bazel, the Pip based custom-op packaging and other misc fixes(?).
Realistically I can't give a timeline for  getting this all integrated back into upstream, purely because the most important parts (public bazel and pip) aren't needed for or tested in the internal build. Are you thinking to to maintain/support your fork + pip package for the medium term anyway, or really wanting to get it upstreamed?

**Comment by ardila** [comment]
@joth76 In the end it wasn't as bad as I thought, once I understood what was going on. Most of the commit history was me futzing around trying to understand things, but once I understood, it was MUCH easier.

Basically all the changes are in this commit: https://github.com/ardila/custom-op/commit/86a7d835f1aeac243a15bc02e1e35a0fe9db272c

I don't have a huge interest in supporting lots of different platforms via pip, mostly because to be honest I don't know how to do that. The wheels I have built now only support certain platforms (but they are common ones like ubuntu 18.04 or whatever it was that I built things on), but I could try to support other folks cloning the repo and building it for themselves on the platform they want to use.

Hopefully the instructions on that repo now are sufficient to build on whatever platform you want to build on. If any future folks are reading this now, feel free to comment on my custom-op repo if you get stuck!
